tests/functional/modules/test_zos_blockinfile_func.py

```diff
@@ -479,7 +479,7 @@ def test_uss_block_insertafter_regex_defaultmarker(ansible_zos_module):
         "block":"ZOAU_ROOT=/mvsutil-develop_dsed\nZOAU_HOME=\\$ZOAU_ROOT\nZOAU_DIR=\\$ZOAU_ROOT",
         "state":"present"
     }
-    full_path = TEST_FOLDER_BLOCKINFILE + inspect.stack()[0][3]
```

Comment:
So happy that you've removed inspect.stack. from naming generation, can you remove the import in line 17 as well :)